livingdoc-fixture-api/src/test/java/org/livingdoc/fixture/api/ExampleScenarioFixture.java

```diff
@@ -0,0 +1,62 @@
+package org.livingdoc.fixture.api;
+
+import java.util.List;
+
+import org.livingdoc.fixture.api.converter.Converter;
+import org.livingdoc.fixture.api.converter.TypeConverter;
+import org.livingdoc.fixture.api.scenario.After;
+import org.livingdoc.fixture.api.scenario.Before;
+import org.livingdoc.fixture.api.scenario.ScenarioFixture;
+import org.livingdoc.fixture.api.scenario.Step;
+
+
+@ScenarioFixture("Example")
+@ScenarioFixture({ "Beispiel", "Ejemplo" })
+@Converter(ExampleScenarioFixture.MovieConverter.class)
+public class ExampleScenarioFixture {
+
+    @Before
+    public void before() {
+
+    }
+
+    @After
+    public void after() {
+
+    }
+
+    @Step("the user '{}' is logged into the shop")
+    @Step("der Benutzer '{}' ist eingeloggt")
+    public void loggedInUser(String username) {
+
+    }
+
+    @Step("he|she also? adds '{}' to his|her shopping cart")
```

Comment:
Yeah, I'think we should build a spike for the Levenshtein distance. For this example I will remove the DSL element.